docs/backtesting.md

```diff
@@ -84,8 +84,8 @@ freqtrade backtesting --timerange=20190501-
 
 #### Advanced use of timerange
 
-Doing `--timerange=20190101-` will all available data starting with January 1st, 2019 from your inputdata.
-You can also specify specific dates, or a range span indexed by start and stop.
+Using `--timerange=20190101-` will use all available data starting with January 1st, 2019 from your inputdata.
```

Comment:
using...will use

"Example: Running backtesting with the `--timerange=20190101-` option will use all available..."?

^ "Example: " (or "For example, ") is missing?